retrieval/resource_map.go

```diff
@@ -117,10 +117,25 @@ var ProxyResourceMap = ResourceMap{
 	},
 }
 
+var AggregatedProxyResourceMap = ResourceMap{
```

Comment:
```suggestion
var ProxyV2ResourceMap = ResourceMap{
```

Call it ProxyV2 would be reasonable for the user as the type is ProxyV2.

Reply:
Done